.stylelintrc.yml

```diff
@@ -1,76 +1,45 @@
 
   # possible errors (these are all on by default)
   no-descending-specificity: null
 
   # limit language features
-  at-rule-no-vendor-prefix: true
   color-named: always-where-possible
-  declaration-block-no-redundant-longhand-properties:
-    - true
-    - ignoreShorthands:
-        - grid-template
-  declaration-block-single-line-max-declarations: 1
+  custom-property-pattern: null
   declaration-no-important: true
   function-url-no-scheme-relative: true
-  media-feature-name-no-vendor-prefix: true
-  property-no-vendor-prefix: true
-  selector-max-empty-lines: 0
-  selector-no-vendor-prefix: true
-  shorthand-property-no-redundant-values: true
-  value-no-vendor-prefix: true
+  selector-class-pattern: null
 
-  # stylistic issues
-  at-rule-name-case: lower
-  at-rule-name-space-after: always
-  at-rule-semicolon-space-before: never
-  color-hex-case: lower
-  color-hex-length: short
-  comment-whitespace-inside: always
-  declaration-bang-space-after: never
-  declaration-bang-space-before: always
-  font-family-name-quotes: always-unless-keyword
-  font-weight-notation: named-where-possible
-  function-max-empty-lines: 1
-  function-name-case: lower
-  function-url-quotes: always
-  indentation: null
-  length-zero-no-unit: true
-  max-empty-lines: 2
-  media-feature-colon-space-after: always
-  media-feature-colon-space-before: never
-  media-feature-name-case: lower
-  media-feature-parentheses-space-inside: never
-  media-query-list-comma-space-after: always
-  media-query-list-comma-space-before: never
-  no-eol-whitespace: true
-  no-missing-end-of-source-newline: true
-  number-leading-zero: always
-  number-no-trailing-zeros: true
-  property-case: lower
-  selector-attribute-brackets-space-inside: never
-  selector-attribute-quotes: always
-  selector-combinator-space-after: always
-  selector-combinator-space-before: always
-  selector-descendant-combinator-no-non-space: true
-  selector-list-comma-newline-after: always
-  selector-pseudo-class-case: lower
-  selector-pseudo-class-parentheses-space-inside: never
-  selector-pseudo-element-case: lower
-  selector-pseudo-element-colon-notation: double
-  selector-type-case: lower
-  unit-case: lower
-  value-keyword-case: lower
-  value-list-max-empty-lines: 1
+  # @@@ Override failing default rules
+  block-no-empty: null
+  color-function-notation: null
+  font-family-name-quotes: null
+  font-family-no-missing-generic-family-keyword: null
+  no-invalid-position-at-import-rule: null
+  number-max-precision: null
+  property-no-unknown: null
+  scss/at-function-pattern: null
+  scss/at-if-no-null: null
+  scss/at-mixin-pattern: null
+  scss/at-rule-conditional-no-parentheses: null
+  scss/comment-no-empty: null
+  scss/dollar-variable-pattern: null
+  scss/function-quote-no-quoted-strings-inside: null
+  scss/function-unquote-no-unquoted-strings-inside: null
+  scss/no-global-function-names: null
+  scss/operator-no-unspaced: null
```

Comment:
@mirisuzanne I disabled all the rules that were throwing errors. It'd probably be better for a Sass person to selectively disable the ones we don't like, and fix the rest of the errors.